selenium/test/test_helper.rb

```diff
@@ -0,0 +1,10 @@
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+require "testcontainers/selenium"
+
+require "minitest/autorun"
+require "minitest/hooks/test"
+require "pry"
```

Comment:
Can you please remove the pry and the dependencies on Gemfile.lock

Reply:
In the pull request of below is already deleted that gem.
Thanks.